packages/affiliates/package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "test": "yarn test-mocha && yarn test-truffle",
```

Comment:
Q: should this also have `yarn test-e2e` or we only want to run them separately to unit tests?

Reply:
currently needs  to not run in CI since it requires BQ auth. Its more meant right now to run locally until we have something in place that doesn't mind connecting to production sources in CI.

Reply:
I think this makes sense. @daywiss and I discussed this, and certain tests that hit apis particularly hard or use gcloud resources directly seem like they may be better suited for nightly runs on master in ci rather than running on every PR. Thoughts?